### ISSUE TITLE
WCAG 2.1 Accessibility features

### ISSUE THREAD

**Issue by r-leyshon** [rstudio/rmarkdown#2176]
Issues found in testing .rmds on [WAVE](https://wave.webaim.org/report#/https://datasciencecampus.github.io/DSCA_WCAG-2.1-Compliance/):

- [ ] author & date are specified as h4, should be h2 with styling in css.
- [ ] no page regions specified. Suggest replacing toc <divs> with <nav>, appropriate use of <main> and so on.
- [ ] no html lang attribute is specified. This can be corrected with `lang: "en"` in the YAML. 
- [ ] very low contrast in chunk output - affects code and code comments. Also applies to the Bootswatch readable theme.

**Comment by r-leyshon** [comment]
For anyone else that find these a priority, I've written a package that helps to avoid some of these 'gotchas' - [accessrmd](https://r-leyshon.github.io/access_rmd_docs/). 1.0.0 now on CRAN.
